common/model-views.cpp

```diff
@@ -557,512 +557,7 @@ namespace rs2
         adjusted_string.replace(pos, to_be_replaced.size(), to_replace);
```

Comment:
Check if this method should remain also here (you have added it also in option-model.cpp.
You may also consider adding it to the option-model class as private method

Reply:
Seems that the method adjust_description is still there - right?